SDKTest/NeuroIDClassTests.swift

```diff
@@ -518,13 +518,21 @@ class NIDNewSessionTests: XCTestCase {
 
     func test_resumeCollection() {
         NeuroID._isSDKStarted = false
+        NeuroID.userID = "temp"
         NeuroID.sendCollectionWorkItem = nil
 
         NeuroID.resumeCollection()
 
         assert(NeuroID._isSDKStarted)
         assert(NeuroID.sendCollectionWorkItem != nil)
     }
+    
+    func test_willNotResumeCollectionIfNotStarted() {
+        NeuroID._isSDKStarted = false
+        NeuroID.resumeCollection()
```

Comment:
should we explicitly set the userID to nil so that there isn't a race issue with other tests?

Reply:
For this doesnt it doesnt need to be nil since the SDK is not started, but I can also set it to nil.